.github/workflows/snapshot-pr-approval.yaml

```diff
@@ -1,21 +1,46 @@
 name: SnapshotsONPRs
 on:
-  push:
-    branches: [ main ]
   pull_request_review:
```

Comment:
maybe triggering on issue_comment too?  The `if` below should short-circuit if the PR is not approved.

Reply:
I think that would become a different Github event i.e. github.event.review.state would not be there anymore, but I'll give it a try